src/IonParserBinaryRaw.ts

```diff
@@ -131,6 +115,17 @@ const ivm_image_1 = IVM.binary[1];
 const ivm_image_2 = IVM.binary[2];
 const ivm_image_3 = IVM.binary[3];
 
+// stack pointer for _ts to store length and type information of container
+class stackPointer {
+  len: number;
+  type: number;
+
+  constructor(len: number, type: number) {
```

Comment:
Subjective: when ordering parameters, I prefer to put the details last. To me, the `type` of a container is more interesting than the number of encoding bytes it occupies, so I'd reverse these. Feel free to ignore this if you disagree.